src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c

```diff
@@ -37,7 +37,7 @@
  *  - loops repeatedly when there is work to do
  *  - moves pending commands into a scheduler command queue
  *
- * [new -> pending]. The xocl API adds exec BOs to KDS.  The exec BOs are
+ * [new -> pending]. The xocl API adds exec BOs to KDS.	 The exec BOs are
```

Comment:
really? what is the rule we're following here?

Reply:
Maybe an editor setting?  I think this came from the merge from Sonal's repo.